src/makeCalculator.js

```diff
@@ -4,7 +4,29 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (a, b) => a + b,
+
+    subtract: (a, b) => a - b,
+
+    multiply: (a, b) => a * b,
```

Comment:
The `multiply` method should update the `result` property. Consider using `this.result *= a * b;` to accumulate the result.